src/libcrun/handlers/krun.c

```diff
@@ -99,7 +99,7 @@ libkrun_exec (void *cookie, libcrun_container_t *container, const char *pathname
 
   ctx_id = krun_create_ctx ();
   if (UNLIKELY (ctx_id < 0))
-    error (EXIT_FAILURE, -ret, "could not create krun context");
+    error (EXIT_FAILURE, 0, "could not create krun context");
```

Comment:
In this case I think we can use `-ctx_id` as errno number:

https://github.com/containers/libkrun/blob/79d993ce4b9a0a959a6c9d3c0017c7eb2f78aa1f/src/libkrun/src/lib.rs#L248-L253

Reply:
thanks, now changed according to suggestion